packages/create-next-app/helpers/copy.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { async as glob } from 'fast-glob'
+import path from 'path'
+import fs from 'fs'
+
+interface CopyOption {
+  cwd?: string
+  rename?: (basename: string) => string
+  parents?: boolean
+}
+
+const identity = (x: string) => x
+
+export const copy = async (
+  src: string | string[],
+  dest: string,
+  { cwd, rename = identity, parents = true }: CopyOption = {}
+) => {
+  const source = typeof src === 'string' ? [src] : src
+
+  if (source.length === 0 || !dest) {
+    throw new TypeError('`files` and `destination` are required')
```

Comment:
Was this code copied from somewhere?

It says `files` in the error message but there is no `files` parameter.

Reply:
It was auto-completed by the copilot when I typed `TypeError` :octocat:.

Fixed in [`04dcc3b` (#53241)](https://github.com/vercel/next.js/pull/53241/commits/04dcc3b773e54e08be5a86974aa204909a67cb0f)